migration/v0.30/apply/13-gatekeeper-psp.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
I see that the manual migration part of the readme is missing these two script steps.